packages/assets-controllers/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Add a new DeFiPositionsController that keeps an update list of DeFi positions for EVM accounts ([#5400](https://github.com/MetaMask/core/pull/5400))
```

Comment:
(we can do this when releasing) - the changelog may need to go into more detail on the new additions.
- the new Controller class. The Actions/Events it takes.
- Any new exported types or code a user can use.

Reply:
Added additional details to the changelog so it's easier for the release.